src/java/org/apache/cassandra/transport/PreV5Handlers.java

```diff
@@ -107,20 +110,16 @@ private void releaseItem(Flusher.FlushItem<Message.Response> item)
             // The only exception to this is if the global request rate limit has been breached, which means
             // we'll have to wait until a scheduled wakeup task unpauses the connection.
             //
-            // note: this path is only relevant when part of a pre-V5 pipeline, as only in this case is
+            // Note: This path is only relevant when part of a pre-V5 pipeline, as only in this case is
             // paused ever set to true. In pipelines configured for V5 or later, backpressure and control
             // over the inbound pipeline's autoread status are handled by the FrameDecoder/FrameProcessor.
             ChannelConfig config = item.channel.config();
 
-            if (!config.isAutoRead())
+            if (!config.isAutoRead()
```

Comment:
could presumably just check `pauseTrigger` in this case? We should maintain that `pauseTrigger != NONE` => `!config.isAutoRead()`

Reply:
Resolved in https://github.com/apache/cassandra/pull/1045/commits/c367d46baeac8cf2e3c1c973fe1f0227d77e2d1a